### PR TITLE
Auto-activate virtual environment in devcontainer

### PR DESCRIPTION
**Pull request by noonfom**
This PR enables auto-activation of the Python virtual environment inside the devcontainer.

Note: #42 should merge to `github-ci` first.

### CODE REVIEW

**Comment by pan-** [comment]
#42 as been merged, can you rebase this PR ?

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -13,6 +13,9 @@ ENV LC_ALL en_US.UTF-8
 RUN apt-get update
 RUN apt-get install -y locales && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 RUN apt-get -fy install vim sudo bash-completion build-essential gdb docker.io
+RUN apt-get update && apt-get install -y valgrind gcovr
+RUN pip3 install virtualenv
+RUN pip3 install mbed-tools --upgrade
```
Comment (pan-) [inline .devcontainer/Dockerfile:18]: How do we get the right version of CMake ?

**Comment by noonfom** [comment]
@pan- rebased

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (noonfom) [reply .devcontainer/Dockerfile:18]: This inherits from Saheer's stable [image](https://github.com/0xc0170/docker-hub/tree/master/mbed-os-env), which has the right CMake version.

File: init.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Reload .bashrc settings
+source ~/.bashrc
```
Comment (paul-szczepanek-arm) [inline init.sh:19]: why?

**Review by paul-szczepanek-arm — APPROVED** [review]

**Review by pan- — APPROVED** [review]

**Pull request merged**